src/vm/prestub.cpp

```diff
@@ -2292,23 +2292,33 @@ EXTERN_C PCODE STDCALL ExternalMethodFixupWorker(TransitionBlock * pTransitionBl
         {
             GCX_COOP_THREAD_EXISTS(CURRENT_THREAD);
 
-            // Get the stub manager for this module
-            VirtualCallStubManager *pMgr = pModule->GetLoaderAllocator()->GetVirtualCallStubManager();
-
-            DispatchToken token;
-            if (pMT->IsInterface())
-                token = pMT->GetLoaderAllocator()->GetDispatchToken(pMT->GetTypeID(), slot);
+            if (pMD != NULL && !pMD->IsInterface())
+            {
+                // v-table based calls
+                pCode = DynamicHelpers::CreateVtableCallHelper(pModule->GetLoaderAllocator(), pMD);
```

Comment:
Since the implementations of CreateVtableCallHelper don't support MethodTable::VTableIndir_t::isRelative, you should check it before calling that function.